src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -2,29 +2,40 @@
 
 set -e
 
+$CLUSTERS=("c1","c2")
+
 start_ceph() {
     cd $FULL_PATH_BUILD_DIR
 
-    MGR=2 RGW=1 ../src/vstart.sh -n -d
-    sleep 10
+    for cluster in ${CLUSTERS[@]}; do
+        MGR=2 RGW=1 ../src/mstart.sh $cluster -n -d
+        sleep 10
```

Comment:
Good chance to remove an active wait (given this will mean 20 seconds for 2 cluster, 30 for 3, etc.). Let's launch it to the background (so that we have the 2 clusters starting in parallel):
```suggestion
        MGR=2 RGW=1 ../src/mstart.sh $cluster -n -d &
```
And then, what about waiting on the `ceph mgr services` with the [`timeout` command](https://stackoverflow.com/a/58759974):
```bash
timeout 60 bash -c 'until ceph mgr services 2>/dev/null | jq -e .dashboard; do sleep 10; done'
```

Reply:
I didn't needed the timeout it seems. and the mstart is kind of synchronous calls. The second one only runs after the first gets completed.

Reply:
Yeah I got what you meant and I started the scripts in parallel. but I can't rely on the `ceph mgr services` cmd where it tries to fetch the dashboard url. because even if this one is succeeded the next one or the rgw command will fail because rgw daemons are not up. It gets up a little later. I was trying different ways up till now to accomplish this task but I couldn't find a neat way to do it.